emoji/src/main/java/com/vanniktech/emoji/EmojiArrayAdapter.java

```diff
@@ -8,21 +8,25 @@
 import android.view.ViewGroup;
 import android.widget.ArrayAdapter;
 import com.vanniktech.emoji.emoji.Emoji;
-import com.vanniktech.emoji.listeners.OnEmojiClickedListener;
+import com.vanniktech.emoji.listeners.OnEmojiClickListener;
+import com.vanniktech.emoji.listeners.OnEmojiLongClickListener;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collection;
 
 import static com.vanniktech.emoji.Utils.checkNotNull;
 
 final class EmojiArrayAdapter extends ArrayAdapter<Emoji> {
-  @Nullable private final OnEmojiClickedListener listener;
-  @Nullable private final OnEmojiLongClickedListener longListener;
+  @Nullable private final VariantEmoji variantManager;
```

Comment:
No, the `EmojiArrayAdapter` of the recent tab gets `null` passed in for the `variantManager`, because we do not want to show what the `variantManager` returns, but that what the `recentManager` returns.

Yes, this is a bit hacky, we could also have a dedicated `EmojiArrayAdapter` for that, but I thought that would be a bit of overkill.